.stylelintrc.json

```diff
@@ -24,7 +24,8 @@
         ],
         "ignoreSelectors": [":export"]
       }
-    ]
+    ],
+    "selector-max-compound-selectors": null
```

Comment:
What about using false

Reply:
This expect an number, array or null
![image](https://user-images.githubusercontent.com/61360270/201177207-b50f8ba5-6ad6-464b-a317-616bb3976bf9.png)